### PR TITLE
Improve performance of goc profile

### PR DESCRIPTION
**Pull request by lyyyuna**

### CODE REVIEW

File: pkg/cover/server.go [hunk 1]
```diff
@@ -26,9 +26,13 @@ import (
 	"net/url"
 	"os"
 	"strconv"
+	"sync"
+
```
Comment (tongjingran) [inline pkg/cover/server.go:30]: Does this empty line needed?

File: pkg/cover/server.go [hunk 2]
```diff
@@ -147,23 +151,47 @@ func profile(c *gin.Context) {
...
+				return
+			}
+			mergedProfiles = append(mergedProfiles, profile)
+		}()
+	}
+
+	wg.Wait()
+	if len(errs) != 0 {
+		for _, err := range errs {
+			log.Warnf("get one profile failed, error: %s", err.Error())
```
Comment (tongjingran) [inline pkg/cover/server.go:189]: Warn logs may need to print immediately when error occurs

**Comment by lyyyuna** [comment]
猜测插桩变量过多会导致 get profile 变慢，试了插桩 10000 个，但并没有复现。

**Pull request closed without merging**